src/main/java/com/epam/tc/hw2/trello/services/MembersService.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.tc.hw2.trello.services;
+
+import static com.epam.tc.hw2.trello.TrelloApi.DOMAIN;
+import static com.epam.tc.hw2.trello.services.BoardsService.BOARDS_ENDPOINT;
+
+import com.epam.tc.hw2.trello.TrelloApi;
+import com.epam.tc.hw2.trello.dto.BoardDto;
+import io.restassured.response.Response;
+import java.util.Arrays;
+
+public class MembersService {
```

Comment:
make a common service 
with common part for this services

Reply:
Done